lib/MooX/Options/Role.pm

```diff
@@ -314,18 +314,15 @@ sub parse_options {
         }
     }
 
-    if (   $opt->help() || defined $params{help}
```

Comment:
$opt is a Getopt::Long::Descriptive::Opts object.  The help attribute only exists if you pass a help option to Getopt::Long::Descriptive::describe_options().  

1. I'm not sure what attributes you want rexistanceemoved from which role?
2. I am checking the existence of the help method on the line after this one.  Perhaps I am misunderstanding something?